.prettierignore

```diff
@@ -0,0 +1 @@
+./src/assets/fonts-awesome
```

Comment:
Could you also include ``**/node_modules/**`` here?

Reply:
Prettier's CLI ignores node_modules by default.

Reply:
Hey @naveen521kk let's remove this since we don't serve `fa` locally anymore. 
Add `./src/assets/` here.